app/src/test/java/com/sethpthomas91/httpserver/ServerRunnerTest.java

```diff
@@ -3,14 +3,52 @@
 import org.junit.Assert;
 import org.junit.Test;
 
+import java.io.IOException;
+
 import static org.junit.Assert.*;
 
 public class ServerRunnerTest {
 
     @Test
-    public void testCasePassingTest() {
-        ServerRunner newServer = new ServerRunner();
-        Assert.assertTrue(newServer.testCasePassingTest());
+    public void testsServerPortDefaultsTo5000() {
+        ServerRunner serverRunner = new ServerRunner();
+        Assert.assertEquals(5000, serverRunner.getPort());
+    }
+
+    @Test
+    public void testsRunServerCanAcceptCustomPort() {
+        ServerRunner serverRunner = new ServerRunner();
+        serverRunner.setPort(5010);
+        Assert.assertEquals(5010, serverRunner.getPort());
     }
 
+    @Test
+    public void testsRunServerStartStartsSocketInsideServerSocketWrapper() throws IOException {
+        ServerRunner serverRunner = new ServerRunner();
+        MockServerSocket mockServerSocketWrapper = new MockServerSocket(5050);
+        serverRunner.setServerSocketWrapper(mockServerSocketWrapper);
+        serverRunner.startServer();
+        Assert.assertTrue(mockServerSocketWrapper.createSocketWasCalled());
+    }
+
+    @Test
+    public void testsRunServerStartStartsSocketReaderInsideServerSocketWrapper() throws IOException {
+        ServerRunner serverRunner = new ServerRunner();
+        MockServerSocket mockServerSocketWrapper = new MockServerSocket(5050);
+        serverRunner.setServerSocketWrapper(mockServerSocketWrapper);
+        serverRunner.startServer();
+        Assert.assertTrue(mockServerSocketWrapper.createReaderWasCalled());
+    }
+
+    @Test
+    public void testsRunServerStartStartsSocketWriterInsideServerSocketWrapper() throws IOException {
+        ServerRunner serverRunner = new ServerRunner();
+        MockServerSocket mockServerSocketWrapper = new MockServerSocket(5050);
+        serverRunner.setServerSocketWrapper(mockServerSocketWrapper);
+        serverRunner.startServer();
+        Assert.assertTrue(mockServerSocketWrapper.createWriterWasCalled());
+    }
+
+
+
 }
```

Comment:
```suggestion
    @Test
    public void testsRunServerStartStartsSocketWriterInsideServerSocketWrapper() throws IOException {
        ServerRunner serverRunner = new ServerRunner();
        MockServerSocket mockServerSocketWrapper = new MockServerSocket(5050);
        serverRunner.setServerSocketWrapper(mockServerSocketWrapper);
        serverRunner.startServer();
        Assert.assertTrue(mockServerSocketWrapper.createWriterWasCalled());
    }
}

```

Reply:
Was this just the spacing issue? I need to get back into using a linter for this.

Reply:
Yep!